instance/tests/models/test_openedx_theme_mixins.py

```diff
@@ -71,7 +74,10 @@ def test_colors_and_images_applied(self):
         OpenEdXInstanceFactory(name='Integration - test_colors_applied', deploy_simpletheme=True)
         instance = OpenEdXInstance.objects.get()
         user = get_user_model().objects.create_user('betatestuser', 'betatest@example.com')
-        self.make_test_application(instance, user)
+        with freeze_time('2019-01-02 09:30:00') as frozen_time:
```

Comment:
@jamestait, wouldn't it be easier to create a class variable named say `FROZEN_TIME` with the time to use for the tests and also have the string-formatted version of it in the class so that this code doesn't have to be repeated in multiple places?

Reply:
To summarise the conversation we had in-person about this, we need `accepted_privacy_policy` to be nullable, as we have existing instances where the user hasn't agreed to the privacy policy. In instances where the field is null, it needs to be editable in the admin, so it can be set on behalf of users. In instances where the field is not null, it needs to be read-only in the admin.

Reply:
> Edit: It looks like this code is out of place in this test and can we find out ways to get rid of it?

Right - it doesn't matter to us here what the timestamp is for the privacy policy acceptance, so we can just update `make_test_application` to provide the current time.